ui/src/components/board-generator/QasBoardGenerator.vue

```diff
@@ -0,0 +1,264 @@
+<template>
+  <qas-grabbable class="qas-board-generator">
+    <div class="no-wrap q-col-gutter-sm q-px-xl row">
+      <div v-for="(header, index) in results" :key="index" class="q-mr-sm">
+        <qas-box class="q-mb-md">
+          <slot :context="header" name="header-column" />
+        </qas-box>
+
+        <div ref="columnContainer" class="qas-board-generator__column">
+          <slot v-for="column in getColumn(header)" :context="column" name="column-item" />
+
+          <div class="full-width justify-center q-mt-sm row">
+            <qas-btn
+              v-if="showSeeMore(getKeyByHeaderKey(header))" icon="sym_r_add" label="Ver mais"
+              :use-label-on-small-screen="false" variant="tertiary" @click="fetchColumn(header)"
+            />
+
+            <q-spinner v-if="columnsLoading[getKeyByHeaderKey(header)]" color="grey-4" size="3em" />
+
+            <qas-empty-result-text v-if="showEmptyResultText(header)" />
+          </div>
+        </div>
+      </div>
+    </div>
+  </qas-grabbable>
+</template>
+
+<script setup>
+import { ref, watch, computed, onMounted, toRaw, inject } from 'vue'
+import promiseHandler from '../../helpers/promise-handler'
+
+defineOptions({ name: 'QasBoardGenerator' })
+
+const columnContainer = ref(null)
+const columnsPagination = ref({})
+const columnsLoading = ref({})
+
+const axios = inject('axios')
+
+const props = defineProps({
+  results: {
+    type: Array,
+    default: () => []
+  },
+
+  columnIdKey: {
+    type: String,
+    required: true
+  },
+
+  columnParams: {
+    type: Object,
+    default: () => ({})
+  },
+
+  columnUrl: {
+    type: String,
+    default: '',
```

Comment:
Acho que num precisa ter o `default` se a prop for `required`

Reply:
feito